src/modules/Elsa.Workflows.Api/Endpoints/WorkflowDefinitions/Dispatch/Endpoint.cs

```diff
@@ -48,22 +38,23 @@ public override async Task HandleAsync(Request request, CancellationToken cancel
             return;
         }
 
-        var instanceId = request.InstanceId ?? _identityGenerator.GenerateId();
-        var correlationId = request.CorrelationId;
-        var triggerActivityId = request.TriggerActivityId;
-        var input = (IDictionary<string, object>?)request.Input;
+        var instanceId = request.InstanceId ?? identityGenerator.GenerateId();
         var dispatchRequest = new DispatchWorkflowDefinitionRequest
         {
             DefinitionId = definitionId,
             VersionOptions = versionOptions,
-            Input = input,
+            Input = (IDictionary<string, object>?)request.Input,
```

Comment:
Since this is an endpoint we should `handle` values besides the expected dictionary as well.

Reply:
Agreed. In this case, since the dispatch request only supports dictionaries, this endpoint should validate the input type, and if it's anything but a dictionary, we respond with a 400 status code. Agreed?